src/elements/common/breadcrumbs/__tests__/Breadcrumbs.test.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+import { shallow } from 'enzyme';
+import { BreadcrumbsBase } from '../Breadcrumbs';
+
+describe('Breadcrumbs', () => {
+    let intl;
+    const getWrapper = (props = {}) => shallow(BreadcrumbsBase(props));
+    const allFilesLocalized = 'localizedValue';
+
+    beforeEach(() => {
+        intl = {
+            formatMessage: jest.fn().mockReturnValue(allFilesLocalized),
+        };
+    });
+
+    afterEach(() => {
+        jest.clearAllMocks();
+    });
+
+    test('should render "All Files" bread crumb in localized string', () => {
```

Comment:
Minor: "breadcrumb," to match the spelling of the component (same comment for `const breadcrumb` below)